packages/amplication-data-service-generator/tsconfig.json

```diff
@@ -18,5 +18,5 @@
     "jsx": "react"
   },
   "include": ["src"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "src/server/static"]
```

Comment:
this was added to resolve this issue:
https://github.com/amit-amp/discord-plugin-test/pull/3/files#diff-e3c8d7e3936df2e438aa9fb770204f3b05c9e3da19e4e7d29ed9fe0036d1260b

I think this is also the reason the CI is failing

Reply:
If we are excluding the entire static folder from TS, we can also remove all @ts-ignore comments.
This is also aligned with the initiative @arielweinberger wanted to promote to avoid all ts-ignore in our templates. 

We don't need to do this as part of this part - but just raising this so @arielweinberger is aware of it.

Reply:
@abrl91 / @arielweinberger, we need to think about how to resolve the CI issue